static/preview.html

```diff
@@ -716,6 +743,35 @@
         } else {
           kernel.authenticate('wss://mainnet.infura.io/ws/v3/f54f2e10b59647778de06d884121f8fa', true)
         }
+
+        if (isElectron()) {
+          await initDesktop()
```

Comment:
if we need to await the initDesktop we should return new Promise<>() at `async function initDesktop`